lib/server.js

```diff
@@ -508,7 +508,15 @@ Object.defineProperties(Server.prototype, {
       } else {
         str = 'ldap://'
       }
-      str += this.host + ':' + this.port
+
+      let host = this.host
+      // Node 18 switched family from returning a string to returning a number
+      // https://nodejs.org/api/net.html#serveraddress
+      if (addr.family === 'IPv6' || addr.family === 6) {
+        host = '[' + this.host + ']'
+      }
```

Comment:
We need a test to check for IPv6 connectivity. Since we don't yet have v18 in the testing matrix, we don't need to cover the branch. I don't want to add v18 to the testing matrix until I can cut the next major release.

Reply:
@MaxLeiter are you able to add this test?

Reply:
Yeah sorry, I misunderstood what you meant by “dont need to cover this test”

Reply:
Test pushed. Lmk if that's what you meant.

Reply:
```
Suites:   ​54 passed​, ​54 of 54 completed​
Asserts:  ​​​2782 passed​, ​of 2782​
```